include/grpc/support/slice.h

```diff
@@ -175,6 +175,10 @@ gpr_slice gpr_empty_slice(void);
 int gpr_slice_cmp(gpr_slice a, gpr_slice b);
 int gpr_slice_str_cmp(gpr_slice a, const char *b);
 
+/* Returns a c-string representation of a slice. Useful for logging or debugging
+   purposes only. The string representation can change in future */
+char *gpr_slice_to_cstring(gpr_slice slice);
```

Comment:
On a second thought, maybe it is better to just remove this function. No one calls it currently and can be added in future if needed. Doesn't feel like its worth going through the pain just for code coverage purposes..